### ISSUE TITLE
Issue navigator shows "Update to recommended settings" after updating Xcode to version 14.3

### ISSUE THREAD

**Issue by Adobels** [yonaskolb/XcodeGen#1349]
After updating Xcode to version 14.3, I noticed that the issue navigator shows an issue with the message "Update to recommended settings". When I click on the details of this issue, I see two sections: "Build settings" and "Project settings".

"Build Settings"
- Target 'Designable' - Enable Module Verifier
Enabling the Apple Clang Module Verifier is recommended for all targets that define clang modules.
"Project Settings
- Enable Parallelization in Command Line Builds Using '-target'
Enable this project setting to allow the build system to build independent targets from this project in parallel when using the '-target' option of 'xcodebuild'.

Could you please provide any guidance on how to resolve this issue?

**Comment by krishsatya** [comment]
Checking the "Parallelize build for command-line builds" checkbox sets `BuildIndependentTargetsInParallel` to `YES` in the Xcode project file. That might be helpful to anyone working on a PR.

**Comment by joridor** [comment]
I was able to suppress the warning by defining the project attributes for both `BuildIndependentTargetsInParallel` and the XCode version (`LastUpgradeCheck`).

```yml
attributes:
  LastUpgradeCheck: "1430"
  BuildIndependentTargetsInParallel: true
```

**Comment by razor313** [comment]
In my case, it has been solved by adding the following lines: 🤷🏻‍♂️
```shell
attributes:
  LastUpgradeCheck: "1430"
```

**Comment by yonaskolb** [comment]
Should be resolved in 2.36.0
